project/src/main/webapp/event_script.js

```diff
@@ -656,44 +715,72 @@ function addDeleteBtn(eventElement, entityType) {
   var date = document.getElementsByName('date')[0].value;
+  var time = document.getElementsByName('time')[0].value;
+  var capacity = document.getElementsByName('capacity')[0].value;
   var description = document.getElementsByName('description')[0].value;
  
   const nameError = document.getElementById('name-error');
+  const locationError = document.getElementById('location-error');
   const dateError = document.getElementById('date-error');
+  const timeError = document.getElementById('time-error');
+  const capacityError = document.getElementById('capacity-error');
   const descriptionError = document.getElementById('description-error');
   const injectionError = document.getElementById('injection-error');
  
   // If comment contains any html of javascript don't submit the form
-  if ((description.includes("<html>")) || (description.includes("<script>")) || (name.includes("<html>")) || (name.includes("<script>"))) {
+  if ((description.includes("<html>")) || (description.includes("<script>")) || (name.includes("<html>")) 
+    || (name.includes("<script>"))) {
+
       injectionError.style.display = "block";
       return false;
   } else {
       injectionError.style.display = "none";
  
   }
   // If name, description, or both are empty don't submit the form
-  if ((description === "") || (name === "") || (date === "")) {
+  if ((description === "") || (name === "") || (date === "") || (time === "") || (capacity === "") 
+    || (location === "")) {
+
       if (description === "") {
-          descriptionError.style.display = "block";
+        descriptionError.style.display = "block";
       } else {
-          descriptionError.style.display = "none";
+        descriptionError.style.display = "none";
       }
       if (name === "") {
-          nameError.style.display = "block";
+        nameError.style.display = "block";
       } else {
-          nameError.style.display = "none";
+        nameError.style.display = "none";
       }
       if (date === "") {
-          dateError.style.display = "block";
+        dateError.style.display = "block";
+      } else {
+        dateError.style.display = "none";
+      }
+      if (time === "") {
+        timeError.style.display = "block";
+      } else {
+        timeError.style.display = "none";
+      }
+      if (capacity === "") {
+        capacityError.style.display = "block";
+      } else {
+        capacityError.style.display = "none";
+      }
+      if (location === "") {
+        locationError.style.display = "block";
       } else {
-          dateError.style.display = "none";
+        locationError.style.display = "none";
       }
       return false;
   } else {
       descriptionError.style.display = "none";
       nameError.style.display = "none";
       dateError.style.display = "none";
+      timeError.style.display = "none";
+      capacityError.style.display = "none";
+      locationError.style.display = "none";
       injectionError.style.display = "none";
   }
```

Comment:
Do you need the outer 
`if ((description === "") || (name === "") || (date === "") || (time === "") || (capacity === "") 
    || (location === "")) {`
?
If they are all filled in, it will accomplish the same as the `else` statement. I feel like I'm missing some intentional purpose here.

Reply:
you can also just do !var instead of var == "", but null and empty string will evaluate to false

Reply:
For Christina: I thought that checking for empty string would be better for readability.

For Fernando: I need the outer if because of the structure of the return statements, I thought it would be better to have one conditional for returning false, and one for returning true.

Reply:
I see, so instead of having a return false in each of the interior ifs, you abstracted that out to the outer if. Sounds good, just checking!